.github/workflows/e2e.yml

```diff
@@ -28,4 +32,13 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Run Playwright tests
-        run: npm run e2e
```

Comment:
I'd rather us using a builder for e2e tests here. I believe we had an example configured that used a generic builder capable of supporting playwright tests? I really don't want to run the app under test using the playwright config as a reference best practice we give to customers and teams

Reply:
This is a good example: https://dev.to/darlanalves/playwright-angular-cli-39ab